pkgs/development/python-modules/python-ndn/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, python3Packages
+, pythonOlder
+}:
+
+buildPythonPackage rec {
+  pname = "python-ndn";
+  version = "0.3-1";
+
+  src = fetchFromGitHub {
+    owner = "named-data";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-+kWhUNu65yQMHP4m/86j0XqrZGrvtkYOLeLKXjzRhpY=";
+  };
+
+  disabled = pythonOlder "3.8";
```

Comment:
It seems that the upstream requires 3.9 or higher.
https://github.com/named-data/python-ndn/blob/1725b13d4c34afe1dca6615b6338bb3edb846c88/setup.cfg#L37

Reply:
argh, typo